shared/ui/build.gradle.kts

```diff
@@ -12,6 +12,7 @@ kotlin {
         val commonMain by getting {
             dependencies {
                 implementation(project(":core:displaymodels"))
+                implementation(project(":core:models"))
```

Comment:
This might be a big import just to get some Id value classes. I wonder if we should put the same value classes on the display models instead? :hmm: